PowerShell/JumpCloud Commands Gallery/commands.json

```diff
@@ -376,5 +383,12 @@
     "link": "https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Commands%20Gallery/Windows%20Commands/Windows%20-%20Software%20Restriction%20Policy.md",
     "description": "Adds a software restriction policy to windows. This policy will restrict the execution of any matching file type in the restricted directory."
+  },
+  {
+    "name": "Windows - Uninstall Remote Assist | v1.0 JCCG",
+    "type": "windows",
+    "command": "$legacyUninstallerPath=\"C:\\Program Files\\JumpCloud\\Jumpcloud Assist App\\Uninstall Jumpcloud Assist App.exe\"\n$uninstallerPath=\"C:\\Program Files\\JumpCloud Remote Assist\\Uninstall JumpCloud Remote Assist.exe\"\n\nif ( Test-Path $legacyUninstallerPath ) {\n    Write-Host \"Uninstalling legacy JumpCloud Remote Assist at \" $legacyUninstallerPath\n    try {\n        $uninstallerProcess = Start-Process -FilePath $legacyUninstallerPath -Wait -PassThru -ArgumentList \"/S\"\n    }\n    catch {\n        Write-Error \"Unable to uninstall legacy JumpCloud Remote Assist\"\n        Write-Error $_\n        exit 1\n    }\n    Write-Host \"Legacy JumpCloud Remote Assist uninstaller completed with exit code $($uninstallerProcess.ExitCode)\"\n}\n\nif ( Test-Path $uninstallerPath ) {\n    Write-Host \"Uninstalling JumpCloud Remote Assist at \" $uninstallerPath\n    try {\n        $uninstallerProcess = Start-Process -FilePath $uninstallerPath -Wait -PassThru -ArgumentList \"/S\"\n    }\n    catch {\n        Write-Error \"Unable to uninstall JumpCloud Remote Assist\"\n        Write-Error $_\n        exit 1\n    }\n    Write-Host \"JumpCloud Remote Assist uninstaller completed with exit code $($uninstallerProcess.ExitCode)\"\n}",
+    "link": "https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Commands%20Gallery/Windows%20Commands/Windows%20-%20Uninstall%20Remote%20Assist.md",
+    "description": "This command will remove the JumpCloud Remote Assist app on a Windows device."
```

Comment:
Shouldn't this be consistent with Mac's (and vice versa), i.e. 
```
Removes the JumpCloud Remote Assist app on a Windows system.
```
?

Reply:
i adjusted the Mac description to be consistent with this